rule/utils.go

```diff
@@ -140,3 +140,81 @@ func isCallToExitFunction(pkgName, functionName string) bool {
 func newInternalFailureError(e error) []lint.Failure {
 	return []lint.Failure{lint.NewInternalFailure(e.Error())}
 }
+
+// isUpper checks if rune is digit
+func isDigit(r rune) bool {
+	return r >= '0' && r <= '9'
+}
+
+// isUpper checks if rune is ascii upper case letter
+func isUpper(r rune) bool {
+	return r >= 'A' && r <= 'Z'
```

Comment:
Ditto

Reply:
Same here, it's normal to do not use unicode.IsUpper

See previous discussion.